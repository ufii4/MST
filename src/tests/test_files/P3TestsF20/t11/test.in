n 5
i 1;2;-1.3
i 1;2;1.4
i 0;3;1.8
edge_count
clear
i 4;5;3.5
degree 4
degree 5
mst
i 3;4;2.6
i 1;2;1.8
i 1;2;0.8
i 0;2;1.4
i 0;1;1.1
i 2;3;0.7
i 1;3;0.6
mst
exit
