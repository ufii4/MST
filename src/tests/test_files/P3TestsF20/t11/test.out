success
invalid argument
success
success
edge count is 2
success
invalid argument
degree of 4 is 0
invalid argument
not connected
success
success
success
success
success
success
success
mst 5.00
